### ISSUE TITLE
Ensure serverless-workflow-examples are working properly

### ISSUE THREAD

**Issue by LuboTerifaj** [apache/incubator-kie-examples#1860]
### Description

Some [serverless-workflow-examples](https://github.com/apache/incubator-kie-kogito-examples/tree/stable/serverless-workflow-examples) may not work properly.

#### Goal
Report individual issues for all examples that do not work properly.
Link all found issues in the comments.

#### Scope
Please mark the example below as checked once it is reviewed and all found issues are reported. Those issues don't have to be resolved as part of this issue.
The goal of this issue is to identify issues in examples. The found issues resolution is out of scope, although welcomed.

#### Examples to be checked

- [x] serverless-workflow-annotations-description	
- [x] serverless-workflow-callback-events-over-http-quarkus	
- [x] serverless-workflow-callback-quarkus	
- [ ] serverless-workflow-camel-routes	
- [ ] serverless-workflow-compensation-quarkus	
- [ ] serverless-workflow-consuming-events-over-http-quarkus	
- [x] serverless-workflow-correlation-quarkus	
- [ ] serverless-workflow-custom-function-knative	
- [x] serverless-workflow-custom-type	
- [x] serverless-workflow-data-index-quarkus	
- [x]  serverless-workflow-data-index-persistence-addon-quarkus
- [x] serverless-workflow-error-quarkus	
- [x] serverless-workflow-events-quarkus	
- [x] serverless-workflow-expression-quarkus	
- [x] serverless-workflow-foreach-quarkus	
- [x] serverless-workflow-functions-events-quarkus	
- [x] serverless-workflow-functions-quarkus	
- [ ] serverless-workflow-funqy	
- [ ] serverless-workflow-github-showcase	
- [x] serverless-workflow-greeting-quarkus	
- [x] serverless-workflow-greeting-rpc-quarkus	
- [x] serverless-workflow-hello-world	
- [ ] serverless-workflow-loanbroker-showcase	
- [x] serverless-workflow-newsletter-subscription	
- [ ] serverless-workflow-oauth2-orchestration-quarkus	
- [ ] serverless-workflow-openvino-quarkus	
- [ ] serverless-workflow-order-processing	
- [ ] serverless-workflow-parallel-execution	
- [x] serverless-workflow-python-quarkus	
- [ ] serverless-workflow-qas-service-showcase	
- [x] serverless-workflow-saga-quarkus	
- [x] serverless-workflow-service-calls-quarkus	
- [x] serverless-workflow-stock-profit	
- [x] serverless-workflow-temperature-conversion	
- [x] serverless-workflow-testing-with-rest-assured	
- [x] serverless-workflow-timeouts-showcase-embedded	
- [x] serverless-workflow-timeouts-showcase-extended	
- [x] serverless-workflow-timeouts-showcase-operator-devprofile

#### Acceptance criteria
All issues are regarding examples are reported  and linked to this issue.

**Comment by LuboTerifaj** [comment]
Found issue in [serverless-workflow-timeouts-showcase-embedded](https://github.com/apache/incubator-kie-kogito-examples/tree/stable/serverless-workflow-examples/serverless-workflow-timeouts-showcase-embedded):
- https://github.com/knative/serving/issues/14803

**Comment by wmedvede** [comment]
For the record, the serverless-workflow-timeouts-showcase-extended example was addressed here https://github.com/apache/incubator-kie-kogito-examples/pull/1857

**Comment by nmirasch** [comment]
@LuboTerifaj, here https://github.com/apache/incubator-kie-kogito-examples/pull/1863 there are some changes on the examples to avoid using the quay.io/kiegroup/kogito-runtime-jvm:latest,  and changing the referenced image at the docker compose by the nightly one to see the latest changes. Even I applied the changes in the serverless-workflow-examples, I just focused my testing cases in mainly working with docker-compose:
- serverless-workflow-callback-quarkus
- serverless-workflow-data-index-persistence-addon-quarkus
- serverless-workflow-data-index-quarkus

Let's see how it goes, once the pr is merged we  can add the check to that ones (at least)

**Comment by nmirasch** [comment]
@LuboTerifaj here #1870 serverless-workflow-newsletter-subscription have been also updated and verified that is working properly

**Comment by LuboTerifaj** [comment]
I found the following issues during review of some examples above:

- https://github.com/apache/incubator-kie-kogito-examples/issues/1945
- https://github.com/apache/incubator-kie-kogito-examples/issues/1946
- https://github.com/apache/incubator-kie-kogito-examples/issues/1947
- https://github.com/apache/incubator-kie-kogito-examples/issues/1947
- https://github.com/apache/incubator-kie-kogito-examples/issues/1948
- https://github.com/apache/incubator-kie-kogito-examples/issues/1949
- https://github.com/apache/incubator-kie-kogito-examples/issues/1950
- https://github.com/apache/incubator-kie-kogito-examples/issues/1951
- https://github.com/apache/incubator-kie-kogito-examples/issues/1952
- https://github.com/apache/incubator-kie-kogito-examples/issues/1953
- https://github.com/apache/incubator-kie-kogito-examples/issues/1954
- https://github.com/apache/incubator-kie-kogito-examples/issues/1955
- https://github.com/apache/incubator-kie-kogito-examples/issues/1956
- https://github.com/apache/incubator-kie-kogito-examples/issues/1957
- https://github.com/apache/incubator-kie-kogito-examples/issues/1958
- https://github.com/apache/incubator-kie-kogito-examples/issues/1959
- https://github.com/apache/incubator-kie-kogito-examples/issues/1960
- https://github.com/apache/incubator-kie-kogito-examples/issues/1961
- https://github.com/apache/incubator-kie-kogito-examples/issues/1962
- https://github.com/apache/incubator-kie-kogito-examples/issues/1963
- https://github.com/apache/incubator-kie-tools/issues/2172
- https://github.com/kiegroup/kogito-docs/issues/8
- https://github.com/kiegroup/kogito-docs/issues/9
- https://github.com/kiegroup/kogito-docs/issues/10
- https://github.com/kiegroup/kogito-docs/issues/11

**Comment by LuboTerifaj** [comment]
Another batch of found issues:
- https://github.com/apache/incubator-kie-kogito-examples/issues/1967
- https://github.com/apache/incubator-kie-kogito-examples/issues/1968
- https://github.com/kiegroup/kogito-docs/issues/12
- https://github.com/apache/incubator-kie-kogito-examples/issues/1969
- https://github.com/apache/incubator-kie-kogito-examples/issues/1970
- https://github.com/apache/incubator-kie-kogito-examples/issues/1971
- https://github.com/apache/incubator-kie-kogito-examples/issues/1972
- https://github.com/apache/incubator-kie-kogito-examples/issues/1973

**Comment by LuboTerifaj** [comment]
Hello @gmunozfe , I am reassigning the issue to you for checking the remaining examples. Please see the description of the issue to understand its goal. Should you have any questions, don't hesitate to reach out to me. Thanks!
